scipy/sparse/tests/test_common1d.py

```diff
@@ -352,10 +352,8 @@ def test_matvec(self, spcreator):
         assert (A @ np.array([1, 2, 3])).shape == ()
         assert Asp @ np.array([1, 2, 3]) == 11
         assert (Asp @ np.array([1, 2, 3])).shape == ()
-        assert (Asp @ np.array([[1], [2], [3]])).shape == ()
```

Comment:
Can we leave these two tests in (but change the resulting shape to `(1,)`?

Actually, they look like exactly the same test. So you can delete the last one, but can you keep the first one with a corrected result?  I tried it locally and it worked.